server/routes/invitation.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const passport = require("passport");
+const Invitation = require("../database/models/invitation");
+const router = express.Router();
+
+router.get("/user/:id/invitations",
```

Comment:
I think this endpoint should be:
`/user/:id`
Otherwise the final endpoint will become
`/invitations/user/:id/invitations`
which seems to be repetitive IMHO since you have already set up this route in app.js as /invitations/